src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -114,6 +126,12 @@ ApiGlobe.prototype.moveLayerDown = function(layer) {
 
 ApiGlobe.prototype.moveLayerToIndex = function(layer, newId) {
     this.scene.getMap().layersConfiguration.moveLayerToIndex(layer, newId);
+    eventLayerChanged.layerId = newId;
+    eventLayerChangedIndex.layerId = newId;
```

Comment:
Called when the index of a layer has changed, return an object with two properties, id and index